resources/default.rb

```diff
@@ -4,19 +4,59 @@
 property :artifactoryonline, :kind_of => String
 property :repository, :kind_of => String, :required => true
 property :repository_path, :kind_of => String, :required => true
+property :artifact_name, :kind_of => String, :required => true
```

Comment:
This is "required" only if "highest=true". Might be better raising some error in case "highest" is set and "artifactory_name" is not set.

Reply:
Added code to after working on review comments. Created separate PR for same.